WordPressKit.xcodeproj/project.pbxproj

```diff
@@ -1910,28 +1912,7 @@
 		93BD27421EE73384002BB00B /* Mock Data */ = {
 			isa = PBXGroup;
 			children = (
-				C92EFF7225E7444400E0308D /* common-starter-site-designs-empty-designs.json */,
-				C92EFF6C25E741E900E0308D /* common-starter-site-designs-malformed.json */,
-				C92EFF6825E7403F00E0308D /* common-starter-site-designs-success.json */,
-				FFA4D4AE2423B33800BF5180 /* wp-forbidden.json */,
-				FFA4D4AF2423B33800BF5180 /* wp-pages.json */,
-				FFA4D4AC2423B1FE00BF5180 /* wp-admin-post-new.html */,
-				F194E1242417EE7E00874408 /* atomic-get-auth-cookie-success.json */,
-				40819784221F74B200A298E4 /* stats-post-details.json */,
-				4081977A221F153A00A298E4 /* stats-visits-day.json */,
-				4081977D221F269A00A298E4 /* stats-visits-month.json */,
-				40819779221F153A00A298E4 /* stats-visits-week.json */,
-				40819770221DFDB600A298E4 /* stats-posts-data.json */,
-				40819774221E497C00A298E4 /* stats-published-posts.json */,
-				404057DB221C9FD70060250C /* stats-referrer-data.json */,
 				937250EB267A15060086075F /* stats-referrer-mark-as-spam.json */,
-				404057D7221C98690060250C /* stats-clicks-data.json */,
-				404057D3221C5FC30060250C /* stats-countries-data.json */,
-				404057CF221C46780060250C /* stats-videos-data.json */,
-				404057CA221B80BC0060250C /* stats-top-authors.json */,
-				40F9880D221ACFB400B7B369 /* stats-streak-result.json */,
-				404057C6221B36070060250C /* stats-search-term-result.json */,
-				984E34F322EF9464005C3F92 /* stats-file-downloads.json */,
```

Comment:
I did not delete these 🤔 . Must've been relocated to different lines since I added one mock data and placed it in the right slot (alphabetically).